### PR TITLE
Dexsearch: Allow searching national dex

### PR DESCRIPTION
**Pull request by Spandamn**
And add descriptions for new parameters

### CODE REVIEW

**Comment by Spandamn** [comment]
image

File: server/chat-plugins/datasearch.js [hunk 1]
```diff
@@ -67,8 +69,10 @@ exports.commands = {
 		`Inequality ranges use the characters '>=' for '≥' and '<=' for '≤', e.g., 'hp <= 95' searches all Pok\u00e9mon with HP less than or equal to 95.`,
 		`Parameters can be excluded through the use of '!', e.g., '!water type' excludes all water types.`,
 		`The parameter 'mega' can be added to search for Mega Evolutions only, the parameter 'gmax' can be added to search for Gigantamax Formes only, and the parameter 'NFE' can be added to search not-fully evolved Pok\u00e9mon that are not in another tier.`,
+		`'Alola' 'Galar', 'Therian', 'Totem', or 'Primal' can be sued as parameters to search for those formes.`,
 		`Parameters separated with '|' will be searched as alternatives for each other, e.g., 'trick | switcheroo' searches for all Pok\u00e9mon that learn either Trick or Switcheroo.`,
 		`You can search for info in a specific generation by appending the generation to ds, e.g. '/ds1 normal' searches for all Pok\u00e9mon that were normal type in Generation I.`,
+		`You can search the National Pokedex by using /nds or by adding natdex as a parameter.`,
```
Comment (Zarel) [inline server/chat-plugins/datasearch.js]: This should say what the difference is between a normal dexsearch.

File: server/chat-plugins/datasearch.js [hunk 2]
```diff
@@ -67,8 +69,10 @@ exports.commands = {
 		`Inequality ranges use the characters '>=' for '≥' and '<=' for '≤', e.g., 'hp <= 95' searches all Pok\u00e9mon with HP less than or equal to 95.`,
 		`Parameters can be excluded through the use of '!', e.g., '!water type' excludes all water types.`,
 		`The parameter 'mega' can be added to search for Mega Evolutions only, the parameter 'gmax' can be added to search for Gigantamax Formes only, and the parameter 'NFE' can be added to search not-fully evolved Pok\u00e9mon that are not in another tier.`,
+		`'Alola' 'Galar', 'Therian', 'Totem', or 'Primal' can be sued as parameters to search for those formes.`,
 		`Parameters separated with '|' will be searched as alternatives for each other, e.g., 'trick | switcheroo' searches for all Pok\u00e9mon that learn either Trick or Switcheroo.`,
 		`You can search for info in a specific generation by appending the generation to ds, e.g. '/ds1 normal' searches for all Pok\u00e9mon that were normal type in Generation I.`,
+		`/dexsearch will search the Galar Pokedex; You can search the National Pokedex by using /nds or by adding natdex as a parameter.`,
```
Comment (Spandamn) [inline server/chat-plugins/datasearch.js]: This good @Zarel?

Comment (Zarel) [reply server/chat-plugins/datasearch.js]: Hm, I wonder if natdex should be default...

Comment (Spandamn) [reply server/chat-plugins/datasearch.js]: Poll in dev room?

**Comment by Spandamn** [comment]
[19:06:07] %Felucia: Spandan I asked this question when I updated dexsearch for gen 8 and was told to put it to galar
[19:07:39] %Felucia: Because dexsearch should generally default to whatever is usable in the primary PS formats

**Comment by Spandamn** [comment]
image
Currently, Totem Formes for Pokemon existing in the Galar dex show illegal for Gen 7. The following commit should fix this

File: server/chat-plugins/datasearch.js [hunk 3]
```diff
@@ -67,8 +69,10 @@ exports.commands = {
 		`Inequality ranges use the characters '>=' for '≥' and '<=' for '≤', e.g., 'hp <= 95' searches all Pok\u00e9mon with HP less than or equal to 95.`,
 		`Parameters can be excluded through the use of '!', e.g., '!water type' excludes all water types.`,
 		`The parameter 'mega' can be added to search for Mega Evolutions only, the parameter 'gmax' can be added to search for Gigantamax Formes only, and the parameter 'NFE' can be added to search not-fully evolved Pok\u00e9mon that are not in another tier.`,
+		`'Alola' 'Galar', 'Therian', 'Totem', or 'Primal' can be sued as parameters to search for those formes.`,
```
Comment (HoeenCoder) [inline server/chat-plugins/datasearch.js]:
```suggestion
		`'Alola' 'Galar', 'Therian', 'Totem', or 'Primal' can be used as parameters to search for those formes.`,
```

**Pull request merged**